cmd/osquery-perf/agent.go

```diff
@@ -1509,6 +1535,7 @@ func main() {
 		randSeed            = flag.Int64("seed", time.Now().UnixNano(), "Seed for random generator (default current time)")
 		startPeriod         = flag.Duration("start_period", 10*time.Second, "Duration to spread start of hosts over")
 		configInterval      = flag.Duration("config_interval", 1*time.Minute, "Interval for config requests")
+		logInterval         = flag.Duration("logger_interval", 1*time.Second, "Interval for scheduled queries log requests")
```

Comment:
nit: specify that this is the interval in which we check to see if any scheduled query needs to run

Reply:
Ah, good point.

Reply:
disregard, but let's name this flag to match the associated `logger_tls_period` config in osquery

Reply:
or at least reference it in the description